arxiv-base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# arxiv/flask base docker file
+
+FROM centos:centos7
+
+RUN yum -y update && yum -y install epel-release \
```

Comment:
Nice use of && chaining and yum clean all to keep layer count and size down while also failing fast, perhaps add a comment noting that.

Reply:
For sake of discussion, multi stage builds or Rocker could mildly simplify this install script creation by ADDing and running it simply to verify package availability then making a second real build stage with just the raw yum install command for readability. https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#use-multi-stage-builds https://github.com/grammarly/rocker

Reply:
Good idea; then one might even use a python script instead of bash! Probably out of scope for now though.